drivers/include/periph/i2c.h

```diff
@@ -388,8 +388,8 @@ int i2c_write_bytes(i2c_t dev, uint16_t addr, const void *data,
  * @brief   Convenience function for writing one byte to a given
  *          register address
  *
- * @note    This function is using a repeated start sequence for writing to the
- *          specified register address.
+ * @note    This function is using a continuous sequence for writing to the
```

Comment:
if you change the note on the write helpers, why not on the  reads (see above), too?

Reply:
Because almost all devices need a repeated start read (you need to write to set the pointer and need to start again to get it into a read state) where as almost all the devices use a continuous write to enter data.  This note reflects implementation now.

Reply:
TD;DR: Because the read is correct...

Reply:
okay